poglink/cogs/rates.py

```diff
@@ -44,6 +45,7 @@ def __init__(self, client):
         self.webpage_urls = client.config.rates_urls
         self.channel_id = client.config.rates_channel_id
         self.polling_delay = client.config.polling_delay
+        self.url_delay = 5
```

Comment:
Added this here for now, but can be made into a config param if needed. I don't expect to need to change this value.

Reply:
Update: I kept it as an instance attribute only so it could be manually overridden during testing.